kuma/javascript/src/payments/pages/index.jsx

```diff
@@ -263,7 +263,7 @@ export default function LandingPage({ data, locale }: RouteComponentProps) {
                                     subscriptionsLink={
                                         <a href={urls.managePayments}>
                                             {gettext(
-                                                'manage monthly subscription page'
+                                                'manage monthly subscriptions page'
```

Comment:
Maybe it is just me 😸 but "subscriptions" suggests that you can have more than one? Perhaps "subscription" is more appropriate in this case?

Reply:
Fixed and searched for all instances of "monthly subscription" to make sure it's singular (except one case where it didn't apply).